src/commands/generate.ts

```diff
@@ -80,7 +80,7 @@ export async function generate(
 
   /** Fetch site metadata. */
   log.info('Fetch site metadata')
-  const siteContext = await parseTable(config.get('url'), notionAgent)
+  const siteContext = await parseTable(config.get('url') as string, notionAgent, config)
```

Comment:
Could you rebase[^1] your branch to the latest `master` branch? Which has improved types for `Config` so you don't need to write `config.get('url') as string` to manually specify the type.

[^1]: In master, `git pull`, go to your branch `git rebase master`.